### DISCUSSION TITLE
Dark Mode Testing

### DISCUSSION THREAD

**Discussion by zackkrida** [WordPress/openverse#4783] [Show and tell]
@WordPress/openverse-maintainers if you enjoy using dark modes, please consider testing Openverse's dark mode implementation at https://staging.openverse.org/?ff_force_dark_mode=on. 

@fcoveram, please also review the dark mode whenever you are able to, and share any of your own feedback and suggestions here.

After some discussion here of the dark mode, we can create issue(s) (if needed) to refine the styling and make changes.

I'd like to open a two-week period for testing these designs, so this discussion will close on August 30th.

**Comment by zackkrida** [comment]
I personally really like this dark mode! One issue I see is that the image loading background color (the "skeleton" loader) is too light in light mode. @fcoveram do you know, is there a darker color that was planned to be used for that?

**Reply by fcoveram** [reply]
I missed adding a dark version of the skeleton. I should also include an update in the layout, the current reflects the previous masonry grid. Thanks for noticing it.

**Comment by kspilarski** [comment]
Hi @zackkrida! I tested dark mode and it's working fine - all pages remained with the same black background color all throughout. 
When searching, it will first display these blank spaces for 2-3 seconds before loading the images and audio: 

<img width="1283" alt="Screenshot 2024-08-19 at 22 35 36" src="https://github.com/user-attachments/assets/aff04ba4-177a-41a7-92db-12ea773491d7">


While testing dark mode, I came across untranslated texts. Let me know if I should open another issue, but I'm sharing my here feedback on that. When I switched the language from English to Polish, not all contents were translated. For example on this page https://staging.openverse.org/pl/sources this  part remained in English, but was translated fully when Deutsch and Espanol were selected: 

> Some providers have multiple different groupings of content within them. [Flickr](https://www.flickr.com/) has sources ranging from NASA to personal photography. The [Smithsonian Institute](https://www.si.edu/) comprises a dozen diverse collections. Wikimedia Commons runs the gamut in terms of content, and is used by several galleries, libraries, archives, and museums highlighting some or all of their digitized collections. 

Polish: 

<img width="1272" alt="Screenshot 2024-08-19 at 22 30 34" src="https://github.com/user-attachments/assets/43d2818c-eb76-4209-ad88-81e337550fd3">

Deutsch: 

<img width="995" alt="Screenshot 2024-08-19 at 22 39 15" src="https://github.com/user-attachments/assets/f9f51830-69b0-417a-9367-ae84d83476e6">

**Reply by zackkrida** [reply]
Thank you so much! Concerning the translations, you should have seen a banner indicating that the translation was partial:

![image](https://github.com/user-attachments/assets/4b123b93-c8c6-421d-be08-97f4e5fe8013)

Do you happen to recall if you dismissed those prior to observing the missing translations? If not, that itself might be an issue.

**Reply by kspilarski** [reply]
I've tried this again in the same browser in private mode and other browsers too, and the banner on top appeared like in your screenshot. Looks like there is no issue on the translation here! Thanks! :)

**Comment by zackkrida** [comment]
I found an additional small issue. Text inputs use the white text so are illegible in dark mode:

![image](https://github.com/user-attachments/assets/6dcbe145-b6ef-42e2-990e-e6e0bc1ab744)

But when selected:

![image](https://github.com/user-attachments/assets/f6e97dec-aec5-46c8-a4ba-508394c149d9)

**Reply by fcoveram** [reply]
All form components come from the WordPress Design Library.

![Form components in Openverse design library](https://github.com/user-attachments/assets/2824dfc0-2b19-4429-8af6-0b7f72d5f11b)

Given that WP library doesn't support dark mode or other themes yet, it seems reasonable to document the ones used in Openverse in the self Design Library. This will allow better documentation and avoid instance breaks caused by updates that Openverse doesn't need.

If the above makes sense, I will make the components on the image as Openverse components, and include the text area to document how it changes in dark mode.

**Comment by josephahaden** [comment]
The highlights when hovering are different for each element on the page. This is a hold over from light mode and for some reason it's not as jarring for me there. If this is a design decision based on modern best practices, then please let that override this opinion. :)  But for example:

the Load More button
<img width="1079" alt="Screenshot 2024-08-20 at 11 24 10 AM" src="https://github.com/user-attachments/assets/9b6884f7-39bd-4dd3-bfff-50066dbd64f7">


the External Sources button
<img width="1079" alt="Screenshot 2024-08-20 at 11 24 27 AM" src="https://github.com/user-attachments/assets/b0f4ef1d-d674-4940-8b9a-4eb2824132e8">

**Reply by zackkrida** [reply]
This one's an implementation bug, the focus rings were designed to be a consistent size and color. Nice catch!

**Comment by kspilarski** [comment]
Hi @zackkrida I have additional feedback on dark mode and sensitive content: 

Including "sensitive content" in your search and clicking that sensitive content from the results to expand it, this message appears. The sensitive content warning is hard to read, with the light grey text against the white background: 

<img width="1245" alt="Screenshot 2024-08-20 at 18 58 43" src="https://github.com/user-attachments/assets/7ffa5c68-8499-4880-a027-34f5a461c468">

Only when you highlight the texts, it's more visible: 

<img width="1254" alt="Screenshot 2024-08-20 at 18 59 17" src="https://github.com/user-attachments/assets/dd311a25-8a69-46e7-8e2e-0aae7fc9a979">

But in light mode, the sensitive content warning is visible with black texts against the white background: 

<img width="1255" alt="Screenshot 2024-08-20 at 19 05 09" src="https://github.com/user-attachments/assets/a38ca90a-b206-460d-a55e-6a7f9ca7973f">

**Comment by sarayourfriend** [comment]
Does the placeholder text inside the search box pass contrast requirements? I can barely see it myself :sweat_smile: 

![image](https://github.com/user-attachments/assets/ce7083d5-fea2-4b78-84a3-f68472dfadf7)

On a small screen, the white background of the analytics banner + the help with translations banner on the homepage will fill a large portion of the screen:

![Screen Shot 2024-08-21 at 08 41 42](https://github.com/user-attachments/assets/af86adf2-0c63-4d26-b7df-37faef8b1ff4)

My network is current too slow for me to load figma to check whether this is intended in the designs, but the background colour of the mobile global audio player's waveform is still white, so the unplayed part of the track's waveform is not visible. (In fact, something is so wrong with my connection at the moment that I cannot upload the screenshot to show you, but I will try again in a bit)

**Reply by sarayourfriend** [reply]
![Screen Shot 2024-08-21 at 08 46 09](https://github.com/user-attachments/assets/35dfdb5b-e43e-4ac8-b887-a5c7beec8850)

**Reply by zackkrida** [reply]
Global audio color is a bug, great catch! The note about having such a white-background-heavy initial loading experience for small-device mobile users of dark mode is a good one.

**Reply by fcoveram** [reply]
> Does the placeholder text inside the search box pass contrast requirements? I can barely see it myself 😅

Yes. The placeholder color is `text-secondary` and it does pass the requirement.

**Reply by fcoveram** [reply]
Sorry, I didn't add enough context to the message above replying to @sarayourfriend's question. In the screenshot, the color contrast looks low, but [the mockups](https://www.figma.com/design/GIIQ4sDbaToCfFQyKMvzr8/Openverse-Design-Library?node-id=3835-10893&m=dev) have the right contrast so it's a bug rather than a wrong design decision

**Reply by sarayourfriend** [reply]
Regarding the search placeholder contrast: could it be because of the pixel density of the display? It looks better on my desktop computer with a very different pixel density:

![image](https://github.com/user-attachments/assets/de78a853-f1e5-4c23-b2d5-93f37f706d1c)

My spouse's laptop is 1080p. When I checked there, it looks somewhere between that and my original screenshot. I haven't measured it in any way, that's just anecdotal. It might also have to do with Firefox and Chrome's different text rendering. It looks pretty different on my desktop in Chrome compared to Firefox. I couldn't see as big a difference on my spouse's laptop in that respect.

So, could it be an issue with the screen I was using at the time, rather than the precise contrast measurement of the text for the search text box? Maybe some of the folks in the a11y channel in WordPress Slack could help clarify this if we need help on it. In particular, I'm not clear on the action item, if there is one, for this problem, especially if it's a contextual issue. What do you feel the solution should be, if any, Francisco? Based on your last comment, I wasn't sure if you intended us to make a change for this.

**Quick edit**: After seeing GitHub render the screenshot at full size, it really looks low contrast to me again. It looks fine on my screen when the pixel density is high, so it really is a problem with this particular combination and low-density displays.

**Reply by fcoveram** [reply]
I still believe it's the color of the text. Here is a comparison of staging and the mockups.

| Staging | Mockup |
|:---|:---|
| ![Zoom in to search input's placeholder text in Openverse homepage](https://github.com/user-attachments/assets/1615525c-836b-4210-a0b3-01a61cb644ee) | ![Search input's placeholder text with accessibility notes](https://github.com/user-attachments/assets/073d4dc9-3d10-4c91-9e33-759b2f12e5da) |

I couldn't reach the color used from the browser inspector, but it should be `text-secondary`. The contrast meets AAA in normal size, so I don't think how useful could be reaching out folks from a11y team given that the requirements is met successfully.

**Reply by sarayourfriend** [reply]
The colour of the placeholder text is set to `text-secondary`, but the weight is different from what Figma is showing. I've noticed that Figma generally uses a slightly heavier rendering of fonts at the same weight. [Figma uses an entirely different rendering algorithm than Firefox or Chrome](https://forum.figma.com/t/why-does-a-font-weight-in-figma-seem-lighter-than-the-same-weight-in-the-browser/2207#why), which can lead to this kind of discrepancy.

If a colour picker isn't finding #a9a9a9 in the live example, it's because [the thinness of the font requires significant aliasing to render smoothly](https://en.wikipedia.org/wiki/Font_rasterization#Types_of_rasterization). There will be lots of in-between values on the edges of the font. [Tailwind has some utility classes for changing the method used to smooth the font, but I don't know if they are relevant here](https://tailwindcss.com/docs/font-smoothing). If the type is too thin, there will inevitably be too small a number of “full coloured” pixels for the contrast to pass. Here is my original screenshot magnified to demonstrate this:

![image](https://github.com/user-attachments/assets/b8d83609-004f-4233-a44d-a4979cb4f3b2)

Your staging screenshot is also different, demonstrating how much variation can exist:

![image](https://github.com/user-attachments/assets/4c9fb3e2-0e73-485e-986c-b9fd04697c8d)

You will notice two differences in these examples based on the font weight rendering as well as the aliasing method used. Your browser/OS uses grey scale aliasing, whereas mine did not (and mine is much harder to read for it). Setting `-moz-osx-font-smoothing: grayscale;` (using the `antialiased` Tailwind class) might fix that if we want to unify the aliasing method used in cases of black-on-white or white-on-black text. The apparent difference in weight is caused by the differences in pixel density of the display that I've mentioned. Magnifying the images shows how much that further complicating things when trying to evaluate whether this combination of colours and weights will be legible in the scenarios we can anticipate and account for.

That's just to say: the colour _instruction_ is technically correct, so there isn't anything to fix if we are keeping the same colour for `text-secondary`. The mitigation needs to come from somewhere else, maybe from increasing the weight. Whether slightly increasing the weight (short of going fully "bold") fully mitigates the issue, I don't know. Changing the colour in addition to the weight would increase the mitigations and cover more scenarios. Forcing grey scale aliasing might also help.

**Reply by fcoveram** [reply]
Very clear explanation. Thanks @sarayourfriend for the thorough comment.

I'm aware of the differences between Figma and browser renderings, so I embrace making the decision from what we see on screens rather than in Figma files. In that vein, we're using `regular (400)` and `semibold (600)` for body texts, and `bold (700)` for headings from `h1` to `h4`.

Regarding possible solutions, I'm drawn to explore forcing the grey scale aliasing first. My concern with switching from semibold to bold is making the whole UI unnecessarily bolder, especially in light theme where the current style looks good (if that happen to be the case for all of us maintainers). In that vein, how does the placeholder in light mode on both the homepage and search results look to you? Both placeholders should have the same style.

What do you think of this @sarayourfriend?

**Reply by sarayourfriend** [reply]
> My concern with switching from semibold to bold is making the whole UI unnecessarily bolder

I share that concern and agree it is good to keep in mind :+1: [The MDN documentation for font-smoothing options indicates that they are not on a standard track, and shouldn't be used because not all users will see them](https://developer.mozilla.org/en-US/docs/Web/CSS/font-smooth?utm_source=devtools&utm_medium=inspector-css-compatibility&utm_campaign=default). [They also link this blog post warning against using font-smoothing settings in CSS and in particular demonstrates that antialiased font smoothing performs poorly on dark-UI text](https://usabilitypost.com/2012/11/05/stop-fixing-font-smoothing/). I am second guessing whether the issue could be helped by changing the smoothing options. My pixel density might not be representative either, despite trying a few different devices.

---

Regarding the light mode versions: I don't think I'd ever looked closely, but they look to have somewhat weak contrast in some situations. The non-focused search header query text is okay because it is bold. However, the "Top {x} results" text suffers, for me.

![image](https://github.com/user-attachments/assets/3c5bc078-7481-41cb-abcf-923dfbc03be9)

The homepage search placeholder is better to me than its dark mode counterpart.

![image](https://github.com/user-attachments/assets/b968b6db-ca9a-456c-a162-e4343bbaf603)

However, I personally strongly prefer light mode UI and text to read, and always find dark mode UIs slightly more difficult to navigate and read. Chock that up to the differences in human visual perception!

---

That being said, I was looking at this again today, and realised that I missed something that should have been really obvious in my previous comment. I had assumed I couldn't get the a9a9a9 colour from the placeholder text due to the aliasing issue, but I only tried on my screenshot. When I try on yours, where there are clear non-aliased pixels, the colour is actually not a9a9a9, but 676767. In other words, you are right that something is incorrect about the colour. I apologise for missing this when looking at it earlier! I looked at the code and was also pretty sure `text-secondary` was being set on the placeholder, and so attributed it to the aliasing issue on my screens. I should have looked closer at your screenshot.

Now that I'm looking at the code again, I'm not sure how I missed it. It's pretty clear that the code also is not setting the colour correctly.

The first problem is that it uses `placeholder-grey-8` instead of `placeholder-text-secondary` or `placeholder-gray-5` which are a9a9a9. Changing it to `text-secondary` does not resolve the issue, because there are also manually defined styles setting `input::placeholder` `color` to 9ca3af. If I force it to a9a9a9, it is definitely better, but maybe not entirely so. I still have some difficulty reading it.

Here are the two side by side from my computer.

![image](https://github.com/user-attachments/assets/a9bfe28b-88d4-4b7f-af35-c9fdb6c51c0f)

The top one is set to a9a9a9. But even pixels that don't look like they are aliased still have changes. The crossbar of the `e` is a9a9a9, but the same area on `a` is a8a8a6. The capital S barely has any pixels that are full a9a9a9. I don't think I am viewing the website at too small a size for my screen, but perhaps it would be helpful to also get a ruler out and see what the "real" size of the text is on our screens. I'm worried now that we could also be focusing on non-representative examples from my screenshots.

**Reply by fcoveram** [reply]
> However, I personally strongly prefer light mode UI and text to read, and always find dark mode UIs slightly more difficult to navigate and read. Chock that up to the differences in human visual perception!

Oh yes. Perception in light and dark can change significantly per person. White lines over dark background tend to "burst" and create a noise in the white spaces in between.

If the correct color does not fix the problem, is it possible to set the font-weight values of regular and bold to a higher level in screens and/or browser settings like yours? Applying this, the UI weight could increase proportionally.

> …but perhaps it would be helpful to also get a ruler out and see what the "real" size of the text is on our screens.

I'm curious about this. Does not 16px render as 16px on every screen?

**Reply by sarayourfriend** [reply]
> If the correct color does not fix the problem, is it possible to set the font-weight values of regular and bold to a higher level in screens and/or browser settings like yours? Applying this, the UI weight could increase proportionally.

I didn't know about this before, but it's possible to write a media query against the pixel density of the display: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/resolution

> I'm curious about this. Does not 16px render as 16px on every screen?

Unfortunately not, neither in the visible size on the screen nor in the literal number of pixels used. MDN describes it this way [on the documentation for the `px` unit](https://developer.mozilla.org/en-US/docs/Web/CSS/length#px):

> For screen displays, it traditionally represents one device pixel (dot). However, for printers and high-resolution screens, one CSS pixel implies multiple device pixels. 1px = 1in / 96.

You're probably already aware of that aspect, that retina displays (high pixel density) use multiple physical pixels to represent a CSS pixel.

Beyond that, though, display dimensions also has a role. Consider two 1080p displays, one a 32-inch display and the other 27-inch. Both have the same number of pixels, but they are spread over different physical dimensions. The 32-inch display has "bigger" pixels than the 27-inch, so the same 16 pixels will appear larger on the 32-inch display than the 27-inch display. It's still 16 pixels on each. Crucially, in this situation, I don't think there's a benefit to readability in the larger display. The displays will render the same exact image, which means the 32-inch display makes the smoothing needed to render fonts much more visible (because the aliased pixels are bigger).

When you go up to a higher-density display like 4k, this becomes awkward because you have to choose some scaling factor. Rendering the screen at 1-to-1 pixels on a 4k display at 32-inches is unreadable small. So then you get into the multiple physical pixels for a single CSS pixel, in addition to the differences in display dimensions. Things start to get weird and hard to predict.

My laptop screen, for example, has a pixel density and size such that 200% scaling leaves everything too cramped. 100% is unreadable. So I have to use something like 150 or 175%. If you look up Hi DPI display scaling issues, you'll see lots of people struggling to find a good balance. My spouse's laptop has a similar size screen (it is slightly wider) but is a 1080p panel. It has much lower pixel density. When I set my computer to 150~175% the same UI elements will roughly match. But my spouse's laptop is using 50 to 75% fewer physical pixels to render those elements. For thin fonts, that means a higher ratio of pixels of a given letter used to smoothen it.

Which I guess all of this is to say that when the contrast of a UI element is "on the edge" of readability, these differences in the physical aspects of the display become more prominent. They get harder to account for, and things get more complicated.

---

I think the best thing to do here would be to just fix the colour issue, and then accept that as good enough for now. With the changes we've made to the way the styling code works, we can accommodate further variations on the UI, like [adding another mode for high-contrast](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-contrast). I enable high contrast designs whenever I can because I struggle to read what is considered "normal" contrast. Unfortunately, this search box, even with the right colour, is difficult for me to read. But I think focusing too much on this specific instance and my experience, if the correct colours pass the contrast test we use, risks over-complicating things too early. 

I propose we fix the colour and confirm it passes the basic contrast test we use, and open an issue to consider an additional high-contrast mode. That way, we can separate the trade-offs that high-contrast requires in UI "boldness" and such things.

If no one else is having an issue with this UI element than me, and if the only solutions are conditional on things like physical screen properties, then I don't think it's worth the complexity. For context: I have a hard time reading many of the UI elements on GitHub _even though I've enabled their high contrast theme_. On GitHub's UI it is very bad, there are large sections of text (like these discussion comments) where it hurts to read unless I zoom the UI in a lot. It isn't unusual for me to come across things like this on websites and applications (Slack is also annoying for this). The Openverse dark mode is pretty much fine everywhere else I've looked. I'd say I spend about as much time in GitHub as I do on the Openverse website, so I don't think it's a difference in familiarity, in this respect.

In other words: one UI element that I personally experience as too-low contrast is not unusual when I interact with contemporary UI. I don't want to get too hung up on this if the ways of addressing it will be overly complex in code.

**Reply by fcoveram** [reply]
Thanks for the explanation. I had no idea about the `@media resolution` property. 

> …The 32-inch display has "bigger" pixels than the 27-inch, so the same 16 pixels will appear larger on the 32-inch display than the 27-inch display.

Interesting. I thought that was already solved by weightening resolution, pixel density, and display size to reach an standardized physical size.

> I propose we fix the colour and confirm it passes the basic contrast test we use, and open an issue to consider an additional high-contrast mode. That way, we can separate the trade-offs that high-contrast requires in UI "boldness" and such things.

+100 to this. A high contrast should be "easy" to design given the alias defined. I wonder if the mix "light/dark mode + high contrast" makes sense. If so, an "Enable high contrast" checkbox sounds sufficient to add in the settings.

**Reply by sarayourfriend** [reply]
> I wonder if the mix "light/dark mode + high contrast" makes sense. If so, an "Enable high contrast" checkbox sounds sufficient to add in the settings.

That's about what I had in mind as well. Here's an issue for high contrast mode: https://github.com/WordPress/openverse/issues/4853.

**Reply by fcoveram** [reply]
Thanks for creating the issue 🙏

**Reply by zackkrida** [reply]
I just wanted to share a massive :heavy_plus_sign: to the proposed solution of a high-contrast mode for the light and dark themes. I am really glad this discussion led to such an idea :smile: 

When I think about reasons why a project like Openverse should exist, and the benefits of 'centralizing' such a solution, as opposed to the innumerable different sources of CC-licensed works out there, trying to create the most accessible and reliable user interface for the largest possible audience ranks high! Frontend development is so increasingly complex and with many nuances that many sites lack the resources and expertise for and I think it's an area we of course still have much room for improvement in, but also an area we can really shine in.

**Comment by fcoveram** [comment]
The implementation looks wonderful. Congratulations 👏  I saw correctly almost everything and just a few minor styles showed up. Here are the thing I found out.

### Incorrect color in focus rings

There are a few elements where the focus ring have the wrong color

| Section | Screenshot |
|:---|:---|
| Header items | ![image](https://github.com/user-attachments/assets/3c6fe6f0-b1e6-4152-b81c-919722643b6b)|
| Clear button in search input in Search header | ![image](https://github.com/user-attachments/assets/b8ea0a7e-097d-458c-a7f4-f729818ffb89)|
| Footer items | ![image](https://github.com/user-attachments/assets/b934ea34-aa63-450a-817f-338203eb7351)|
| Source in audio details page | ![image](https://github.com/user-attachments/assets/38605fab-b067-44bc-92ed-0e7fa8b45aa3)|

### Focus rings with white halo

The elements listed below have a white ring in addition to the primary one. I wonder if we should remove it instead of applying the `bg` color as over other colored background the error will still exist.

| Section | Screenshot |
|:---|:---|
| Image result in All content results | ![image](https://github.com/user-attachments/assets/af7e2691-7391-4cf6-8d23-79baf46b9bcb)|
| Audio result in All content results | ![image](https://github.com/user-attachments/assets/f09d52e0-f7ed-4699-bc09-d5a145a27421)|
| Links in body content in all internal pages | ![image](https://github.com/user-attachments/assets/a892e854-e6a1-4ff6-b162-441d7117d535)|
| Link in filter safe browsing section | ![image](https://github.com/user-attachments/assets/f286756e-34e3-42eb-b305-7684c3fc9276)|
| Link to license in media details page |  ![image](https://github.com/user-attachments/assets/958cf4fc-ea76-4b2f-94fe-a5d19374eef7)|
| Links in Attribution box | ![image](https://github.com/user-attachments/assets/61c7e111-d04a-45c2-9b15-1815cadac4a9)|

### Incorrect focus ring

The toggle has an incorrect ring

![image](https://github.com/user-attachments/assets/ce068cc2-9470-479e-bcb9-420d99c25c3a)

### Dark snackbar

The snackbar remained dark instead of changing to its lighter version. There is also a white ring that might relate to a section above.

![image](https://github.com/user-attachments/assets/553453a1-ad8a-4ec5-967a-07e02b67b4f2)

### Hover style of search input

The search input changes the background color when it should just change the placeholder color. This is incorrect in both themes, yet reporting it to address it in parallel.

![image](https://github.com/user-attachments/assets/77ebc129-bb0e-4cc6-871d-caf17e133ea5)

### Audio control is not visible

When focusing the wave area to jump between seconds, the control is not visible.

![image](https://github.com/user-attachments/assets/555cb489-b351-4bbf-8870-c3114fb8a696)

### The machine-generated thumbnail doesn’t follow the theme

The designs don't have this documented, but based on the color scheme defined, the dark dots should be `bg` color.

![image](https://github.com/user-attachments/assets/97f6d0dc-689a-4618-961b-814cb0700f79)

### Disabled style in button

The pointer seems to announce that the report button is disabled. If this is correct, then the disabled style is incorrect in both modes. Reporting it here as a note for addressing it in the future.

![CleanShot 2024-08-26 at 16 57 40](https://github.com/user-attachments/assets/21b4ddb3-a6a9-439e-830e-11ef5ca43089)

**Reply by zackkrida** [reply]
This all looks excellent @fcoveram.

As for the report button, it is disabled until the required 20 characters are typed in the textarea.

**Reply by fcoveram** [reply]
Yes. Looking at the gif again, I noticed the "Report" button is slightly bigger. I will create an issue pointing out the necessary changes.

**Comment by zackkrida** [comment]
Thank you all for the feedback! Testing is over and the suggestions have been moved into a new issue: #4911
